### ISSUE TITLE
Tachiyomi & j2k 1 page transition issue

### ISSUE THREAD

**Issue by SaneZer0** [Jays2Kings/tachiyomiJ2K#1491]
### Steps to reproduce

I understand that, the page needs to load hence it shows the transition. (Transition disabled)

What's the issue Its 1 singular page, you can physically see the page is loaded. And not until you scroll down further it gets removed and by the time you scroll down further. It removes itself.

![Screenshot_20230111-035059.png](https://user-images.githubusercontent.com/65837094/211713759-e80706ff-097c-4016-8aae-b707184427df.png)

![Screenshot_20230111-035154.png](https://user-images.githubusercontent.com/65837094/211713864-026d9ebc-01da-4dde-897e-422db667621c.png)

### Expected behavior

Even though you can see page (its only 1 page) it still shows it, not until you scroll down further it removes it self.

### Actual behavior

N/A

### TachiyomiJ2K version

1.6.0

### Android version

Android 13

### Device

Google Pixel 6 pro 

### Other details

This is an issue on both J2k & tachiyomi as well.

I've tried to create a bug report about this issue with tachiyomi but they blocked it for me to clarify what's the actual issue is. And I'm bringing it here wondering if this could be an bug or just a feature that can't be necessarily fixed in j2k.

https://github.com/tachiyomiorg/tachiyomi/issues/8828

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Tachiyomi](https://github.com/tachiyomiorg/tachiyomi)
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.6.0](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by TheGoldenSun** [comment]
Wait, are you trying to say that some of the pages are cut off while transitioning to the next page? 

Damn it's no wonder tachiyomi isn't responsible for this.
